src/ucp/api/ucp.h

```diff
@@ -251,9 +251,28 @@ enum ucp_ep_close_mode {
                                               for all endpoints created on
                                               both (local and remote) sides to
                                               avoid undefined behavior. */
-    UCP_EP_CLOSE_MODE_FLUSH         = 1  /**< @ref ucp_ep_close_nb schedules
+    UCP_EP_CLOSE_MODE_FLUSH         = 1, /**< @ref ucp_ep_close_nb schedules
                                               flushes on all outstanding
                                               operations. */
+    UCP_EP_CLOSE_MODE_STREAM_SYNC   = 2  /**< @ref ucp_ep_close_nb will flush
+                                              the local endpoint and handle the
```

Comment:
Need to reword, maybe something like this:
`@ref ucp_ep_close_nb will close the stream connection. The remote side will have a @ref UCP_STREAM_POLL_FLAG_CLOSED flag for the remote endpoint if @ref ucp_stream_worker_poll is called on the remote worker...`